src/antsibull_docs/augment_docs.py

```diff
@@ -60,6 +60,30 @@ def add_full_key(options_data: t.Mapping[str, t.Any], suboption_entry: str,
                 _full_keys=full_keys_k)
 
 
+def _add_seealso(seealso: t.List[t.MutableMapping[str, t.Any]],
+                 plugin_info: t.Mapping[str, t.Mapping[str, t.Any]],
+                 ) -> None:
+    for entry in seealso:
+        if entry.get('description'):
+            continue
+        plugin = ''
+        plugin_type = 'module'
+        if entry.get('module'):
+            plugin = entry['module']
+        elif entry.get('plugin') and entry.get('plugin_type'):
+            plugin = entry['plugin']
+            plugin_type = entry['plugin_type']
+        if (
+            plugin and plugin_type in plugin_info and plugin in plugin_info[plugin_type]
+            and isinstance(plugin_info[plugin_type][plugin].get('doc'), dict)
+            and plugin_info[plugin_type][plugin]['doc'].get('short_description')
+        ):
+            desc = plugin_info[plugin_type][plugin]['doc']['short_description']
```

Comment:
This feels a bit ugly to me. Would something like (

``` python
try:
    desc = plugin_info[plugin_type][plugin]['doc']['short_description']
except (KeyError, TypeError):
    continue
```

) be better?

Reply:
That would work, but then an additional `if desc:` would be needed to avoid `None` / `''` to be used.

Reply:
I've implemented that in af06c768fc88bb01d19b6192d544fa8cd19d6555.

Reply:
This actually introduced a bug. The problem is that `plugin_info` can be a defaultdict, whence this construct modifies it when `plugin_type` is not present. #169 fixes this.